### ISSUE TITLE
paper-behavior accessibility issues 

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/paper-behaviors#14]
Audit results: 

[Warning] Text elements should have a reasonable contrast ratio (3)Not applicable tests (7)Passing tests (7)

The disabled button text is too light. I know this is tough because it's trying to get the point across that it's disabled, but can we work on boosting that contrast? 

I am unable to get to the radio button using ChromeVox on Chrome Os, VoiceOver on Mac, or NVDA on Chrome/Windows. I can get to it using TAB, but not the screen readers. 
On Firefox with NVDA on Windows, I can access it, but all the spoken feedback I get is "clickable, clickable". This needs to be fixed. 

Windows specific: 
Firefox / NVDA: Between the buttons and the heading below them, I hear "blank".

**Comment by blasten** [comment]
@lpalmaro I was using VoiceOver on Mac to get to the radio button. I used the `VO keys (ctrl + option) + left`  It also got to it using TABs.  FYI, I added a title to the radio that will say `Radio button with focus state`.  Is there a way to get to the radio button that we haven't covered yet?

**Comment by lpalmaro** [comment]
Nope, I tested this last night as well and on Mac with VO, this does work
fine now. I'm going to do a quick test on Windows to make sure it's working
there, and then we can close this out.

**Comment by lpalmaro** [comment]
Hey Alice,

When you have a chance would you mind updating the demos again?

Now across platforms, I see a different UI for this demo, so I'm wondering
if it's just outdated now?

Screenshot:

[image: Inline image 1]

**Comment by lpalmaro** [comment]
Works fine. Good to close on my end!

**Comment by blasten** [comment]
Thanks
